shared/featureconfig/config.go

```diff
@@ -105,6 +105,42 @@ func InitWithReset(c *Flags) func() {
 	return resetFunc
 }
 
+// Copy returns copy of the config object.
+func (c *Flags) Copy() *Flags {
```

Comment:
Is it possible to test this function? We could easily miss adding new flags to it.